package.json

```diff
@@ -16,9 +16,11 @@
     "typescript",
     "react-native"
   ],
+  "peerDependencies": {
+    "react-native": "^0.41.2 || ^0.42.3"
```

Comment:
I'm running React Native `0.42.3` and it's complaining otherwise as the peer dependency is not matched. We may also do something like `>= 0.41.0  <= 0.42.3` which we could easily upgrade depending on RN versions